js/alert-box.js

```diff
@@ -14,28 +14,69 @@ bcpl.alertBox = (($) => {
 		sessionStorage.setItem('isAlertDismissed', true);
 	};
 
-	const init = () => {
+	const renderAlertBox = (alertData) => {
+		const alertsTemplateHtml = $('#alerts-template').html();
+		const $alertsTarget = $('#alerts-target');
+
+		if (alertsTemplateHtml && alertsTemplateHtml.length) {
+			const alertsTemplate = Handlebars.compile(alertsTemplateHtml);
+
+			if (alertData && alertData.IsEmergency) {
+				alertData.EmergencyClass = 'emergency'; // eslint-disable-line no-param-reassign
+			}
+
+			$alertsTarget.html(alertsTemplate({ alertData }));
+		}
+
+		displayNotificationBar(!alertData);
+	};
+
+	const getAlertDescription = (callback) => {
+		if (callback && typeof callback === 'function') {
+			$.ajax(CONSTANTS.baseApiUrl + CONSTANTS.shared.urls.alertNotification)
+				.then(
+					(data) => data ? callback(data, true) : callback(undefined, false),
```

Comment:
would consider making these methods and then passing them in (this is my preference but not a deal breaker)
```javascript
const onSuccess = () => { /* stuff goes here */ };
const onFailture  = () => { /* stuff goes here */ };
...
.then(onSuccess, onFailure);
```

Reply:
Done, though since I'm passing through a callback, I still had to use function bodies.